test/private/notifications.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { notifications } from '../../src/private/notifications';
+import { Utils } from '../utils';
+import { _uninitialize } from '../../src/public/publicAPIs';
+import { NotificationPayload } from '../../src/public/interfaces';
+
+describe('logs', () => {
+  // Use to send a mock message from the app.
+  const utils = new Utils();
+  const notificationPayload: NotificationPayload =  {
+    payload : "{ \"eventId\" : 1500,  \"environment\" : \"Production\"}", 
+    subPath : 'shifts'
```

Comment:
Is this '/shifts' or just 'shifts'?

Reply:
its actually /shifts..I'll correct it although its in tests.